src/actions/build.rs

```diff
@@ -229,4 +239,30 @@ impl BuildAction<'_> {
             ..Default::default()
         })
     }
+
+    fn parse_contracts_names(&self) -> Vec<String> {
+        match &self.contracts_names {
+            Some(string) => remove_extra_spaces(string)
+                .map(|string| {
+                    string
+                        .split(' ')
+                        .map(ToString::to_string)
+                        .collect::<Vec<_>>()
+                })
+                .unwrap_or_else(|_| {
+                    Error::FailedToParseArgument("contracts_names".to_string()).print_and_die()
+                }),
+            None => vec![],
+        }
+    }
+}
+
+fn remove_extra_spaces(input: &str) -> Result<String, &'static str> {
+    // Ensure there are no other separators
+    if input.chars().any(|c| c.is_whitespace() && c != ' ') {
```

Comment:
Isn't it always false?

Reply:
There are a few other whitespaces https://doc.rust-lang.org/reference/whitespace.html, but of course, it is doubtful to see something other than `' '`. `\n` is whitespace and it wouldn't compile if you pass `erc20\nerc721`